src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs

```diff
@@ -490,5 +347,31 @@ private void Dispose(bool disposing)
                 fsWatcher.Dispose();
             }
         }
+
+        public class Startup
```

Comment:
This shows the pattern for service injection and configuration for the host/SDK, also shows usage of the `Add/UseWebJobsScriptHost*` methods.